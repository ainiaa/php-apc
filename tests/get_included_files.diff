031+ Warning: Wrong parameter count for get_included_files() in /home/huixinchen/opensource/pecl/apc/tests/get_included_files.php on line 21
031- Warning: get_included_files() expects exactly 0 parameters, 1 given in %s on line %d
034+ [Thu Sep  6 18:11:27 2012]  Script:  '/home/huixinchen/opensource/pecl/apc/tests/get_included_files.php'
035+ /home/huixinchen/opensource/pecl/apc/apc_zend.c(38) :  Freeing 0x12030968 (4 bytes), script=/home/huixinchen/opensource/pecl/apc/tests/get_included_files.php
036+ === Total 1 memory leaks detected ===