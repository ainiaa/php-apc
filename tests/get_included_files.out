*** Testing get_included_files()

-- List included files at start --
array(1) {
  [0]=>
  string(65) "/home/huixinchen/opensource/pecl/apc/tests/get_included_files.php"
}

-- List included files atfter including inc1 -
array(2) {
  [0]=>
  string(65) "/home/huixinchen/opensource/pecl/apc/tests/get_included_files.php"
  [1]=>
  string(70) "/home/huixinchen/opensource/pecl/apc/tests/get_included_files_inc1.inc"
}

-- List included files atfter including inc2 which will include inc3 which includes inc1 --
array(4) {
  [0]=>
  string(65) "/home/huixinchen/opensource/pecl/apc/tests/get_included_files.php"
  [1]=>
  string(70) "/home/huixinchen/opensource/pecl/apc/tests/get_included_files_inc1.inc"
  [2]=>
  string(70) "/home/huixinchen/opensource/pecl/apc/tests/get_included_files_inc2.inc"
  [3]=>
  string(70) "/home/huixinchen/opensource/pecl/apc/tests/get_included_files_inc3.inc"
}

-- Error cases --

Warning: Wrong parameter count for get_included_files() in /home/huixinchen/opensource/pecl/apc/tests/get_included_files.php on line 21
NULL
===DONE===
[Thu Sep  6 18:11:27 2012]  Script:  '/home/huixinchen/opensource/pecl/apc/tests/get_included_files.php'
/home/huixinchen/opensource/pecl/apc/apc_zend.c(38) :  Freeing 0x12030968 (4 bytes), script=/home/huixinchen/opensource/pecl/apc/tests/get_included_files.php
=== Total 1 memory leaks detected ===